docs/fabrication/hand-tools/chain-breakers.md

```diff
@@ -0,0 +1,15 @@
+# Chain Breaks
+## Parts of a Chain
+### Outer Plate
+### Inner Plate
+### Roller
+### Pins
+## Using Chain Breaks
+## Chain Break Functions
+### Breaking Chain Links
+### How to Use Chain Breaks
```

Comment:
How to use chain breaks and Using Chain breaks is the same thing, no need to have another section for it